### PR TITLE
34 enum and list

### PR DESCRIPTION
**Pull request by JoeTessier**

### CODE REVIEW

File: arondor-common-reflection-gwt/src/main/java/com/arondor/common/reflection/gwt/client/CssBundle.java [hunk 1]
```diff
@@ -100,6 +100,12 @@
 
         @ClassName("doConversionBtn")
         String doConversionBtn();
+
+        @ClassName("paddL40")
+        String paddL40();
```
Comment (Doloops) [inline arondor-common-reflection-gwt/src/main/java/com/arondor/common/reflection/gwt/client/CssBundle.java]: Could we find a more descriptive name than paddL40 ?

File: arondor-common-reflection-gwt/src/main/java/com/arondor/common/reflection/gwt/client/nview/NListView.java [hunk 2]
```diff
@@ -12,13 +15,24 @@
...
 {
+    private static final Logger LOG = Logger.getLogger(NListView.class.getName());
+
     private final MaterialButton addButton = new MaterialButton();
 
+    // private final FlowPanel inputGroupPanel = new FlowPanel();
+
+    public NListView()
+    {
+        // LOG.severe(">> asking to create a listView");
```
Comment (Doloops) [inline arondor-common-reflection-gwt/src/main/java/com/arondor/common/reflection/gwt/client/nview/NListView.java]: We probably do want to display a pretty explicit error message here, at least as long as the class is not implemented.

File: arondor-common-reflection-gwt/src/main/java/com/arondor/common/reflection/gwt/client/view/ImplementingClassView.java [hunk 3]
```diff
@@ -88,23 +86,24 @@ public void onClick(ClickEvent event)
...
-        });
-        implementingListInput.addMouseOutHandler(new MouseOutHandler()
-        {
-            @Override
-            public void onMouseOut(MouseOutEvent event)
-            {
-                implementingListInput.clearHelperText();
-            }
-        });
+        // implementingListInput.addMouseOverHandler(new MouseOverHandler()
```
Comment (Doloops) [inline arondor-common-reflection-gwt/src/main/java/com/arondor/common/reflection/gwt/client/view/ImplementingClassView.java]: Remove commented lines please !

**Review by Doloops — COMMENTED** [review]
Small comments on code cleanliness, nothing blocking.

File: arondor-common-reflection-gwt/src/main/java/com/arondor/common/reflection/gwt/client/CssBundle.java (hunk 1 above)
Comment (JoeTessier) [reply arondor-common-reflection-gwt/src/main/java/com/arondor/common/reflection/gwt/client/CssBundle.java:105]: standing for **Padding Left : 40 [px]**, sounds pretty descriptive to me, doesn't it ?

**Pull request merged**